sdk/contentsafety/azure-ai-contentsafety/pom.xml

```diff
@@ -43,7 +43,9 @@
   </developers>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jacoco.skip>true</jacoco.skip>
+    <jacoco.min.linecoverage>0.4</jacoco.min.linecoverage>
+    <jacoco.min.branchcoverage>0.3</jacoco.min.branchcoverage>
+    <jacoco.skip>false</jacoco.skip>
```

Comment:
Do we need to set `<jacoco.skip>false</jacoco.skip>`, I though default is `false`?

Reply:
No, we don't. Not technically an error though...

Reply:
removed